component/src/main/java/com/celements/calendar/search/DefaultEventSearchQuery.java

```diff
@@ -19,7 +21,7 @@
 
   private final WikiReference wikiRef;
   private final LuceneQuery luceneQuery;
-  private final List<String> sortFields;
+  private List<String> sortFields = Collections.emptyList();
```

Comment:
```suggestion
  private List<String> sortFields = ImmutableList.of();
```

Reply:
Collections.emptyList is already immutable. Furthermore it is a singleton.